src/components/FortyHours.vue

```diff
@@ -17,6 +17,11 @@
           </li>
         </ul>
       </div>
+       <ul>
+        <li v-for="(tempassignee, assignee) in tempassignee" v-bind:key="assignee">
+           {{ assignee }}: total Hours: {{tempassignee }} 
```

Comment:
you do not need new UL for that use above one

Reply:
you need to write it in same <ul><li> as current, previous

Reply:
done